torchtune/models/llama3/_model_builders.py

```diff
@@ -62,7 +62,25 @@ def llama3_70b() -> TransformerDecoder:
         rope_base=500000.0,
     )
 
-
+def llama3_1_405b() -> TransformerDecoder:
```

Comment:
I think this should be removed - this is in the Llama3 folder, not 3.1

Reply:
great catch - this was inadvertent.  Removing!